regulations/tasks.py

```diff
@@ -72,10 +74,16 @@ def submit_comment(self, body, files):
     except (ClientError, RequestException) as exc:
         logger.exception(exc)
         raise self.retry(exc=exc)
+    except Task.MaxRetriesExceededError as exc:
+        message = "Exceeded retries, saving failed submission"
+        logger.exception(message)
```

Comment:
Minor: the `exception` method on loggers is meant to be called with an exception object. Otherwise, it's the same as the `error` method.